### ISSUE TITLE
How about your SLAM ATE?

### ISSUE THREAD

**Issue by Epiphany-f** [PeterFWS/Structure-PLP-SLAM#21]
I run your SLAM code in my ubuntu18.04,and i find it is not good about translation of ATE.You can watch my picture,it is in point and line mode.
![微信图片_20240116163109](https://github.com/PeterFWS/Structure-PLP-SLAM/assets/58239558/5faef59a-d6d0-4dd3-aff8-0cae0ee1c45a)

**Comment by PeterFWS** [comment]
Hi,

are you able to reconstruct similar point-line maps as I illustrated in the examples (e.g. MH04)? if not, probably something is wrong.

Best,

**Comment by Epiphany-f** [comment]
Yeah, I can reconstruct point-line maps when i run MH04.I don't change your code.Finally I find the camera is reversed when i run MH04,it is strange.

**Comment by Epiphany-f** [comment]
Are you chinese?

**Comment by PeterFWS** [comment]
> Yeah, I can reconstruct point-line maps when i run MH04.I don't change your code.Finally I find the camera is reversed when i run MH04,it is strange.

okay, if the camera is reversed then it is the problem... I am not sure why it happened actually, I guess there is a bug.

Best,

**Comment by PeterFWS** [comment]
> Are you chinese?

yes.

**Comment by Epiphany-f** [comment]
> > Yeah, I can reconstruct point-line maps when i run MH04.I don't change your code.Finally I find the camera is reversed when i run MH04,it is strange.
> 
> okay, if the camera is reversed then it is the problem... I am not sure why it happened actually, I guess there is a bug.
> 
> Best,
请问一下，我跟你跑的这个数据集的结果相差很大吗？因为我没怎么改代码，我就是跑一下的你的结果看看。

**Comment by PeterFWS** [comment]
> > > Yeah, I can reconstruct point-line maps when i run MH04.I don't change your code.Finally I find the camera is reversed when i run MH04,it is strange.
> > 
> > 
> > okay, if the camera is reversed then it is the problem... I am not sure why it happened actually, I guess there is a bug.
> > Best,
> > 请问一下，我跟你跑的这个数据集的结果相差很大吗？因为我没怎么改代码，我就是跑一下的你的结果看看。

2 年前在我的 PC 上跑的结果类似，这些结果在paper附录中。 在升级Ubuntu后我确实发现....甚至换了一台新笔记本电脑，相机也颠倒了......我不知道bug在哪里。你可以看看我上传到github的视频。 线检测,跟踪, 和BA 的实现可能对你有用，你可以尝试重新实现它。谢谢。

**Comment by Epiphany-f** [comment]
> > > > Yeah, I can reconstruct point-line maps when i run MH04.I don't change your code.Finally I find the camera is reversed when i run MH04,it is strange.
> > > 
> > > 
> > > okay, if the camera is reversed then it is the problem... I am not sure why it happened actually, I guess there is a bug.
> > > Best,
> > > 请问一下，我跟你跑的这个数据集的结果相差很大吗？因为我没怎么改代码，我就是跑一下的你的结果看看。
> 
> 2 年前在我的 PC 上跑的结果类似，这些结果在paper附录中。 在升级Ubuntu后我确实发现....甚至换了一台新笔记本电脑，相机也颠倒了......我不知道bug在哪里。你可以看看我上传到github的视频。 线检测,跟踪, 和BA 的实现可能对你有用，你可以尝试重新实现它。谢谢。

好的，有机会我会修复，谢谢您的回复与帮助。

**Comment by rangingwaves** [comment]
@Epiphany-f 修复了没有，大佬，我的相机也是倒过来的
